Tools/runtests.ps1

```diff
@@ -23,4 +23,4 @@ if ($selectedVsVersion -eq '') {
 
 $testFiles = $tests | % { ".\BuildOutput\Release$selectedVsVersion\Tests\$_.dll" }
 
-& "$vstest" @args $testFiles
+& "$vstest" /settings:$(".\Build\default." + $selectedVsVersion + "Exp.testsettings") @args $testFiles
```

Comment:
Just noticed this script won't work if you're running from a different directory. For convenience, let's use something like $PSScriptRoot to establish filepaths.